oeps/processes/oep-0021-proc-deprecation.rst

```diff
@@ -275,15 +277,47 @@ Do the following to document your proposal:
       **Deprecation** section. See Deprecated_ for considerations.
    #. If automated migration will be needed, explain your migration plan in the
       **Migration** section.
-   #. If there is any additional publicly shareable information or data from
-      your earlier analysis, include that in the **Additional Info** section.
+   #. **Additional Info**:
+
+      - If there is any additional publicly shareable information or data from
+        your earlier analysis, include that in the **Additional Info** section.
+      - If you know who plans to take ownership of the ticket, include the
+        name in the **Additional Info** section. **Important**: the deprecation
+        ticket will not move forward until someone takes ownership.
 
 #. Check that your ticket appears on the `DEPR Project Board`_.
 
 .. note::
   You can create a ticket even if you don't yet have all the information
-  gathered yet. You simply must wait until you have completed the issue before
-  moving on to communicating.
+  gathered yet. You simply must wait until you have completed the issue and
+  have an owner before moving on to the formal **Communication** step.
+
+(Optional) Start a discussion
+-----------------------------
+If you want to start a discussion or seek an owner for the deprecation ticket,
+you can post a message in the `Development Discussion Board`_ and any
+appropriate channels in the `Open edX slack`_.
+
+.. _Development Discussion Board: https://discuss.openedx.org/c/development/11
+
+Take Ownership
+--------------
+The person taking ownership of the ticket is not necessarily the person doing
```

Comment:
In the Product Review Process we have a role called a Coordinator. Perhaps that language would make more sense than "owner"? https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Roles

Reply:
@dianakhuang, @feanil, does this work for you?

Reply:
So it would be  along the lines of: "All active DEPRs must have a coordinator." and "The coordinator is responsible for communicating if they're unable to continue coordinating a deprecation."  I think that's fine with me.

Reply:
changes made, along with some rewording.